--- origsrc/novprog-3.0.0/src/data.cpp	2014-10-25 09:15:19.000000000 -0500
+++ src/novprog-3.0.0/src/data.cpp	2015-04-27 15:33:23.894354300 -0500
@@ -378,8 +378,8 @@ void Database::updateValues()
 	double delta = remaining / days;
 	int start = m_start_value - ((m_start_value / 10000) * 10000);
 	for (int i = 1; i <= count; ++i) {
-		m_data[Total].minimum_values.append(std::round(delta * i) + start);
-		m_data[Daily].minimum_values.append(std::round(remaining / days));
+		m_data[Total].minimum_values.append(round(delta * i) + start);
+		m_data[Daily].minimum_values.append(round(remaining / days));
 		days -= 1;
 		int value = m_data[Daily].values.value(i - 1);
 		if ((i >= end) && (value == 0)) {
